Doc/library/ipaddress.rst

```diff
@@ -793,6 +800,43 @@ Interface objects
       :class:`IPv4Interface`.
 
 
+Operators
+^^^^^^^^^
+
+Interface objects support some operators.  Unless stated otherwise, operators
+can only be applied between compatible objects (i.e. IPv4 with IPv4, IPv6 with
+IPv6).
+
+
+Logical operators
+"""""""""""""""""
+
+Interface objects can be compared with the usual set of logical operators.
+
+For equality comparison (``==`` and ``!=``), both the IP address and network
+must be the same for the objects to be equal.  An interface will not compare
+equal to any address or network object.
+
+For ordering (``<``, ``>``, etc) the rules are different.  Interface and
+address objects with the same IP version can be compared, and the address
+objects will always sort before the interface objects.  Two interface objects
+are compared by comparing their networks, using the same rules as
+:class:`IPv4Network` or :class:`IPv6Network`.  The IP address plays no part in
```

Comment:
Not true any more. Changed in commit 7bd8d3e794782582a4ad1c9749424fff86802c3e. IP address takes part in comparison and it's no longer strange.

Reply:
I've removed that and added a `:versionchanged:` for this new comparison.